### PR TITLE
Primitive value multiargs

### PR DESCRIPTION
**Pull request by PalumboN**
VM: https://github.com/pharo-project/pharo-vm/pull/763/

Support meta-compilation of primitives with variadic args (_multiargs_):

Using the pragma like this:
```st
<numberOfArguments: #(0 1)>
```

Druid will compile a generator with the pattern: 
```st
	s1 := self methodNumArgs.
	s1 = 0 ifTrue: [ ... ]

	s2 := self methodNumArgs.
	s2 = 1 ifTrue: [ ... ]

        ^ 0
```

And also leave the entry in the primitiveTable without arguments (`-1` at the end).

------

In this PR I also:

- Removed the `<argumentCountCheck>` customization -> we can have the same behaviour with these changes
- Fix some cases of Phi Functions with Null due to bad selection of the possible new operands under block splitting.
- Fix many tests, I left skipped the more complicated (I'll open issues for that).

### CODE REVIEW

**Comment by PalumboN** [comment]
Some related tests are broken, I'll check them.

**Comment by PalumboN** [comment]
Tests are fixed (or most of them). We need to merge VM: https://github.com/pharo-project/pharo-vm/pull/763 before.

File: Druid-Tests/DRDeadPathEliminationTest.class.st [hunk 1]
```diff
@@ -85,7 +84,7 @@ DRDeadPathEliminationTest >> testEliminateDoubleDeadPaths [
 	| cfg lastBlocks lastPaths lastPathsLeft lastPathsRight |
 	cfg := self generateDruidIRFor: #primitiveDNA.
 
-	3 timesRepeat: [ 
+	4 timesRepeat: [
```
Comment (guillep) [inline Druid-Tests/DRDeadPathEliminationTest.class.st:87]: cuack :)

File: Druid/DRVMState.class.st [hunk 2]
```diff
@@ -213,7 +213,7 @@ DRVMState >> simplify [
 { #category : #accumulating }
 DRVMState >> stackOperandAt: stackPosition [
 
-	^ operandStack at: stackPosition
+	^ operandStack at: stackPosition ifAbsent: [ DRNullValue new ]
```
Comment (guillep) [inline Druid/DRVMState.class.st:216]: This could be dangerous

**Review by guillep — APPROVED** [review]
Ok!

**Pull request merged**